agent/agent.go

```diff
@@ -160,13 +178,20 @@ func (a *Agent) registerConsulService() error {
 	consulService := &consul.AgentServiceRegistration{
 		ID:   a.cfg.InstanceName,
 		Name: a.cfg.ServiceName,
-		Tags: []string{"console-agent"},
+		Tags: []string{"trento-agent"},
 		Checks: consul.AgentServiceChecks{
 			&consul.AgentServiceCheck{
 				CheckID: "ha_checks",
 				Name:    "HA config checks",
 				Notes:   "Checks whether or not the HA configuration is compliant with the provided best practices",
-				TTL:     a.cfg.TTL.String(),
+				TTL:     a.cfg.CheckerTTL.String(),
+				Status:  consul.HealthWarning,
+			},
+			&consul.AgentServiceCheck{
```

Comment:
I'm not really sure why the discovery part goes in the Checks. To report if the discovery failed?

Reply:
Yes, it helps troubleshooting discovery failures. for now there isn't a fail that is being reported, but it would make sense to use the framework for that going forward.